docs/source/contributing_plugins.rst

```diff
@@ -0,0 +1,115 @@
+Contributing plugins
+--------------------
```

Comment:
I think it's worth linking to https://redhat-cop.github.io/automation-good-practices/#_plugins_good_practices in this section.

Reply:
+2 for this. I discovered the repo yesterday because it was linked in an ansible-lint issue. A worth reading for every edpm-ansible contributor.